addons/main/functions/fnc_isInCrew.sqf

```diff
@@ -0,0 +1,7 @@
+#include "script_component.hpp"
+if (is3DEN || !hasInterface) exitWith { false };
+
+params [["_player", objNull]];
+private _vehicle = vehicle _player;
+
+(_vehicle != _player) && { fullCrew [_vehicle, "cargo"] findIf {_player == _x # 0} == -1 }
```

Comment:
Please change the `#` to `select`. The linter hates this as `#` is a pre processor thing.
```suggestion
(_vehicle != _player) && { fullCrew [_vehicle, "cargo"] findIf {_player isEqualTo (_x select 0)} isEqualTo -1 }
```